### PR TITLE
overlap email on video player

### PR DESCRIPTION
**Pull request by Dhroov7**
https://github.com/coding-blocks/motley/pull/289 
Motley PR related to this PR.

### CODE REVIEW

**Comment by Dhroov7** [comment]
@abhishek97 @witty123 @vdvibhu20 Please review.

File: app/pods/components/hls-player/template.hbs [hunk 1]
```diff
@@ -21,7 +21,39 @@
...
+
+  video{
+    z-index: 0;
+  }
+
+</style>
+
+<script>
+
+  let overlay = document.getElementById("overlay")
```
Comment (abhishek97) [inline app/pods/components/hls-player/template.hbs]: Excuse me?
Move these to appropriate places.

File: app/pods/components/hls-player/template.hbs [hunk 2]
```diff
@@ -21,7 +21,39 @@
...
 {{/if}}
-<video id="video" class="w-100" controls onended={{onVideoCompleted}}></video>
+  <video id="video" class="w-100 video-overlay" controls onended={{onVideoCompleted}}></video>
+  <div class="overlay-content">
+    <p class="grey">{{currentUser.user.email}}</p>
+  </div>
+
+<style>
+
+  video{
```
Comment (witty123) [inline app/pods/components/hls-player/template.hbs]: move this to elements in motley

**Comment by Dhroov7** [comment]
@witty123 @abhishek97 Removed the style tag and moved them in motley but what about the JS?

**Comment by abhishek97** [comment]
move to appropriate lifecycle hook.

**Pull request merged**